cost-analyzer/values.yaml

```diff
@@ -536,6 +536,40 @@ kubecostFrontend:
 #    fqdn: kubecost-multi-diag.kubecost.svc.cluster.local:9007
 #  clusterController:
 #    fqdn: cluster-controller.kubecost.svc.cluster.local:9731
+#
+
+  # Configurable headers for nginx responses.
+  nginxHeaders:
+    # applied to all route locations
+    server:
+      - Content-Security-Policy "default-src 'self' cdn.userway.org; frame-ancestors 'none';";
```

Comment:
This looks like it'd be a new header. I'm not super familiar with CSPs, do we know this will work?

Reply:
No, I still need to test it 😅 

The idea is that sources like scripts, images, styles, etc can only be loaded from the same domain the UI is being accessed through, and cdn.userway.org. And also that the application cannot be embedded in an iframe. MDN has great CSP documentation if you feel like digging in more to check my understanding

Reply:
Just pushed an update that I have tested, and it actually does work now.

I had to include directives for `font-src data:;` and I added `unsafe-inline` to `default-src`. I'm not sure whether these will still be flagged. It's safer than having no CSP, and we have something, somewhere, that is adding inline styles to the document.